pinot-core/src/main/java/org/apache/pinot/core/accounting/PerQueryCPUMemAccountantFactory.java

```diff
@@ -155,6 +159,20 @@ protected PerQueryCPUMemResourceUsageAccountant(PinotConfiguration config, boole
       _isThreadCPUSamplingEnabled = isThreadCPUSamplingEnabled;
       _isThreadMemorySamplingEnabled = isThreadMemorySamplingEnabled;
       _isThreadSamplingEnabledForMSE = isThreadSamplingEnabledForMSE;
+      _isPerQueryMemoryCheckEnabled = config.getProperty(
```

Comment:
If you add these config params to `QueryMonitorConfig` - they can be changed at runtime.